packages/components/src/date-time/style.scss

```diff
@@ -5,6 +5,8 @@
 /*rtl:end:ignore*/
 
 .components-datetime {
+	padding: $grid-size;
```

Comment:
Is this needed for all datetime components even outside the modal context? I feel this is specific to the editor usage.

Reply:
You are right. This component that is included is horrendously messy with arbitrary inline styles. It needs a fresh approach, so I've committed some work in progress and will try a separate PR that goes at it differently.

![Screenshot 2019-11-15 at 09 37 41](https://user-images.githubusercontent.com/1204802/68929228-3ab55680-078c-11ea-9d1f-89c75e0378ce.png)

![Screenshot 2019-11-15 at 09 37 35](https://user-images.githubusercontent.com/1204802/68929231-3d17b080-078c-11ea-8a02-816dfcc69390.png)

┻━┻ ︵ヽ(`Д´)ﾉ︵﻿ ┻━┻o

Reply:
But it is a good question, though, should it come with a padding or not?

The original version of this PR suggested that _yes_, this component should be born with padding, and if you need the component inline, as in the post-publish, you should override it there. Because otherwise you use the vanilla Popover component with the vanilla DateTime component, and you get this:

![Screenshot 2019-11-15 at 09 46 50](https://user-images.githubusercontent.com/1204802/68929633-05f5cf00-078d-11ea-876e-7db018b58dc5.png)

I can remove some negative margins to fix the horizontal scrollbar:

![Screenshot 2019-11-15 at 09 49 48](https://user-images.githubusercontent.com/1204802/68929791-4e14f180-078d-11ea-9341-10e421481d3d.png)

But the question becomes, if the datetime component should not have padding, should the popover?

I'm going to try something different, but this is context for why I added the padding.

Reply:
What's causing the horizontal scrollbar in the first place is this:

![Screenshot 2019-11-15 at 09 58 34](https://user-images.githubusercontent.com/1204802/68930389-8d900d80-078e-11ea-9159-bba5c1f84b2d.png)

And it is virtually impossible to refactor that width away because the entire datetime component referenced is reliant on this.